census/src/main/java/module-info.java

```diff
@@ -22,6 +22,7 @@
  */
 module org.openjdk.skara.census {
     requires org.openjdk.skara.xml;
+    requires static org.openjdk.skara.forge;
```

Comment:
What does `requires static` mean in a `module-info.java` file?

Reply:
It's the java modules way of declaring a dependency optional. It's only needed at compile time, but not at runtime. 

At least in this case, the dependency is only used in a static method, which means any types from the forge module would only be used if you call that static method, and to be able to do that, you would already have had to load those classes.

I don't think it matters in Skara currently, as I think all modules that require census also require forge, but it feels a bit cleaner to me that pulling in forge is optional here.